base/irrationals.jl

```diff
@@ -22,7 +22,9 @@ abstract type AbstractIrrational <: Real end
     Irrational{sym} <: AbstractIrrational
```

Comment:
Instead of this see also link, how about we update all of these in another PR to  take the form:
```suggestion
    Irrational{sym} <: [`AbstractIrrational`](@ref)
```

Reply:
Does this work? On a toy example, the REPL prints the link out verbatim, rather than just highlighting the text. 

If it could be made to work, though, it would be good to have globally.

Reply:
oh right, the indentation makes it a code block, and code blocks can't have links in them

We would need to go through and convert them all into code fences such as:
```suggestion
`Irrational{sym}` <: [`AbstractIrrational`](@ref)
```